packages/server/src/koaJsonRpc/index.ts

```diff
@@ -52,7 +52,8 @@ export default class KoaJsonRpc {
 
   constructor(logger: Logger, register: Registry, opts?) {
     this.koaApp = new Koa();
-    this.limit = '1mb';
+    const limit = process.env.INPUT_SIZE_LIMIT || 1;
+    this.limit = limit + 'mb';
```

Comment:
Kind of weird that `this.limit` can be overwritten if you send in `opts` and it has a `limit` below in line 62.  We should probably only use one potential mechanism for setting this configuration value.

Reply:
Yes, I agree with @lukelee-sl